src/electron-runner.js

```diff
@@ -94,6 +94,24 @@ function createWindow() {
 		}
 	} );
 
+	if ( process.platform === 'darwin' ) {
+		const { systemPreferences } = electron;
+		const osTheme = systemPreferences.isDarkMode() ? 'dark' : 'light';
+		window.webContents.executeJavaScript( `
+			window.localStorage.setItem( 'osTheme', '` + osTheme +
```

Comment:
Given this is a template string, interpolation could be used instead of the `+`:
```suggestion
			window.localStorage.setItem( 'osTheme', '${ osTheme }
```